editor.planx.uk/src/@planx/components/FindProperty/Public/index.tsx

```diff
@@ -119,7 +136,11 @@ function Component(props: Props) {
         setRegions([...new Set(regions)]);
       }
     }
-  }, [data]);
+
+    if (geojson?.type) {
```

Comment:
Is there a reason we need to check `type` and not just `geojson`?

Reply:
nope - this is a remnant of an early thought that we might need to check for edge casey stuff like council boundaries that are multipolygons here - but `if (geojson)` should be fine here at this stage :+1: